dimes/dimes.py

```diff
@@ -85,7 +85,7 @@ class TimeSeriesPlot:
     """Time series plot."""
 
     def __init__(self, time_values: list):
-        self.fig = Figure()
+        self.figure = Figure()
         self.time_values = time_values
         self.subplots: List[Union[TimeSeriesSubplot, None]] = [None]
```

Comment:
I am trying to understand why self.subplots can be equal to [None]. Does this mean that if there are no subplots, then self.subplots is [None]? Why do you want to have a list of TimeSeriesSubplot and None?

Reply:
TimeSeriesSubplots are instantiated in the `add_time_series` method when a new subplot_number is specified. We cannot guarantee that time series have been added to all subsequent subplot_numbers. So we need a way to indicate that a subplot has not been instantiated yet (hence `None`).